content/work/top-dog.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Top Dog
+sub: Bet on live sports with play money
+logo: topdog
+client: &client Top Dog
+date: 2020-06-01
+end: 2021-04-01
+image:
+  src: work/top-dog/topdog.jpg
+  alt: three lists of games to bet on including NFL and NHL
+  position: top
+tags:
+  - Sports Sector
+people:
+  - &griffin
+    name: Griffin Parker
+    face: james-weber.jpg
+    url: https://www.linkedin.com/in/jamessw/
+    title: Founder
+    venue: *client
+press:
+  - text: |
+      @@@
+    <<: *griffin
+    slug: configurations
```

Comment:
Can't merge with placeholder data. These need to be fleshed out or removed.

Reply:
Exactly. (See comment above.)